### DISCUSSION TITLE
Question about updating README.rst.

### DISCUSSION THREAD

**Discussion by ShaneLudwig01** [obsproject/obs-studio#10757] [General]
Hi, sorry to bother everyone. I did not realize that Discussions is somewhere I could post this at. But I noticed that the README.rst was missing a direct link to the Code of Conduct. I didn't know where to make a documentation issue link at, as the New Issue Link looked like it was meant purely for Bug reports or crashes. I went and noticed that when making a pull request, it has a template for Documentation, so I filled that out instead. I'm making this discussion basically to ask if you still want an issue link for my pull request or if this is fine. Here is a link to my [pull request](https://github.com/obsproject/obs-studio/pull/10753) following the template given.

**Comment by Fenrirthviti** [comment]
PRs do not require a linked issue, it is only if there is a related issue already to ensure linking is done so that the issue is autoclosed when the PR is merged.

**Reply by ShaneLudwig01** [reply]
I see, thank you very much.
